### ISSUE TITLE
Enlarge the font in the tab bar

### ISSUE THREAD

**Issue by colourfreak** [AlphaWallet/alpha-wallet-ios#3769]
<img width="1361" alt="Screenshot 2022-01-21 at 10 15 18" src="https://user-images.githubusercontent.com/51817359/150500330-887b7eea-d1bf-4677-8e73-87021e405aba.png">

Make it 13pt.

Zeplin
https://zpl.io/aNWMXkD

```
const wallet = {
  width: 94,
  height: 22,
  fontFamily: "SourceSansPro",
  fontSize: 13,
  fontWeight: "600",
  fontStyle: "normal",
  lineHeight: 22,
  letterSpacing: 0,
  textAlign: "center",
  color: colors.AZURE
};
```

```
const activity = {
  width: 94,
  height: 22,
  fontFamily: "SourceSansPro",
  fontSize: 13,
  fontWeight: "normal",
  fontStyle: "normal",
  lineHeight: 22,
  letterSpacing: 0,
  textAlign: "center",
  color: colors.DOVE
};
```

**Comment by eviltofu** [comment]
I'm using these two lines but somehow only the `.normal` text attribute is being used. The selected text attribute is being ignored.
```swift
    UITabBarItem.appearance().setTitleTextAttributes([.font: Style.TabBar.Font.normal], for: .normal)
    UITabBarItem.appearance().setTitleTextAttributes([.font: Style.TabBar.Font.selected], for: .selected)
```
This is somehow due to `UINavigationController` being the view controllers in `UITabBarController`. I created a new project with just `UIViewControllers` in `UITabBarController` and the different font styles do appear. Once I embed the view controller into a `UINavigationController`, the font styles only show the ones for `.normal`. I've tried setting the `tabBarItem` for all the `UINavigationController` and the `UIViewController` but this doesn't work. Any clues on how to proceed?

@oa-s @hboon

**Comment by oa-s** [comment]
@eviltofu try this one
```
lazy var tabBarController: UITabBarController = {
        let tabBarController = TabBarController()
        tabBarController.tabBar.isTranslucent = false
        tabBarController.delegate = self

        let tabBarAppearance = UITabBarAppearance()
        tabBarController.tabBar.tintColor = Colors.appTint
        tabBarAppearance.shadowImage = UIImage(color: Style.TabBar.Separator.color, size: CGSize(width: 0.25, height: 0.25))
        tabBarAppearance.backgroundImage = UIImage(color: Style.TabBar.Background.color)
        let tabBarItemAppearance = UITabBarItemAppearance()

        tabBarItemAppearance.normal.titleTextAttributes = [.font: Style.TabBar.Font.normal ]
        tabBarItemAppearance.selected.titleTextAttributes = [.font: Style.TabBar.Font.selected]

        tabBarAppearance.stackedLayoutAppearance = tabBarItemAppearance
        
        tabBarController.tabBar.standardAppearance = tabBarAppearance
        if #available(iOS 15.0, *) {
            tabBarController.tabBar.scrollEdgeAppearance = tabBarAppearance
        } else {
            // Fallback on earlier versions
        }

        return tabBarController
    }()
```

**Comment by eviltofu** [comment]
> @eviltofu try this one
> 
> ```
> lazy var tabBarController: UITabBarController = {
>         let tabBarController = TabBarController()
>         tabBarController.tabBar.isTranslucent = false
>         tabBarController.delegate = self
> 
>         let tabBarAppearance = UITabBarAppearance()
>         tabBarController.tabBar.tintColor = Colors.appTint
>         tabBarAppearance.shadowImage = UIImage(color: Style.TabBar.Separator.color, size: CGSize(width: 0.25, height: 0.25))
>         tabBarAppearance.backgroundImage = UIImage(color: Style.TabBar.Background.color)
>         let tabBarItemAppearance = UITabBarItemAppearance()
> 
>         tabBarItemAppearance.normal.titleTextAttributes = [.font: Style.TabBar.Font.normal ]
>         tabBarItemAppearance.selected.titleTextAttributes = [.font: Style.TabBar.Font.selected]
> 
>         tabBarAppearance.stackedLayoutAppearance = tabBarItemAppearance
>         
>         tabBarController.tabBar.standardAppearance = tabBarAppearance
>         if #available(iOS 15.0, *) {
>             tabBarController.tabBar.scrollEdgeAppearance = tabBarAppearance
>         } else {
>             // Fallback on earlier versions
>         }
> 
>         return tabBarController
>     }()
> ```
> 
> `lazy var tabBarController: UITabBarController = {` its `InCoordinator.swift`, line `161`

@oa-s Ok, this works. Thanks!
